pkg/kubeapiserver/options/authentication.go

```diff
@@ -605,6 +608,8 @@ func (o *BuiltInAuthenticationOptions) ToAuthenticationConfig() (kubeauthenticat
 				return kubeauthenticator.Config{}, fmt.Errorf("failed to set up public service account keys: %w", err)
 			}
 			ret.ServiceAccountPublicKeysGetter = keysGetter
+		} else if o.ServiceAccounts.ExternalPublicKeysGetter != nil {
```

Comment:
similar comment about making KeyFiles and ExternalPublicKeysGetter mutually exclusive?

```go
switch {
case len(o.ServiceAccounts.KeyFiles) > 0 && o.ServiceAccounts.ExternalPublicKeysGetter != nil:
  ... error
case len(o.ServiceAccounts.KeyFiles) > 0:
  ...
case o.ServiceAccounts.ExternalPublicKeysGetter != nil:
  ...
}
```

Reply:
Done